app/views/shared/comment/_show_partial.html.erb

```diff
@@ -0,0 +1,25 @@
+<% present comment do |presenter| %>
+
+  <%= presenter.text %>
+  <%= presenter.attribution_link %>
+
+  <div class="vote_wrapper_comment">
+    <div class="vote_count_comment">
+      <%= presenter.sum %>
+    </div>
+    <%= render partial: "shared/vote/presenter_index",
+                        locals: { parent: comment } %>
+  </div>
+
+  </br >
+
+  <% if presenter.replies? %>
+    <div class="replies-posted">
+      <% presenter.replies.each do |reply_comment| %>
+        <%= render partial: "shared/comment/show_partial",
```

Comment:
just a minor nitpick, but partials aren't usually named with the suffix of `_partial`. I'd consider renaming this to `comment/_details` or something like that to give it a bit more of a description of what's in it instead of how it's used, if that makes sense.